index.html

```diff
@@ -183,8 +343,11 @@ <h3 id="about-me"class="motto">
   <button id="about-me"class="resume" type="button">
     Get my resume
   </button>
+</div>
+<div class="bottom">
```

Comment:
- [ ] Please I highly recommend sticking to the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34-848&t=Hq64bUbN7WFDPuHZ-0). 😃 

_**REQUIRED FIGMA DESIGN**_


![Screenshot (200)](https://user-images.githubusercontent.com/78104544/230512030-17531733-282a-4a0d-b8fc-e3c8093eaf5d.png)

_**YOUR DESIGN**_

- [ ] Please I highly recommend adding gaps to the `bottom` class list items from the `arrow-down` icon and `arrow-next` icons per the Figma Design. 😃  

![Screenshot (201)](https://user-images.githubusercontent.com/78104544/230512319-f980d07d-08d7-401a-9047-a17244cdab19.jpg)